draft-ietf-tokbind-https-08.xml

```diff
@@ -222,7 +222,7 @@
 
     <abstract>
       <t>This document describes a collection of mechanisms that allow
-      HTTP servers to cryptographically bind authentication tokens
+      HTTP servers to cryptographically bind authentication or authorization tokens
```

Comment:
so to put it all together, the suggestion is to say..

"This document describes a collection of mechanisms that allow HTTP servers to cryptographically bind security tokens (such as cookies and OAuth tokens) ..."